middle_end/flambda/from_lambda/cps_conversion.ml

```diff
@@ -325,76 +523,119 @@ let rec_catch_for_for_loop
                 Lstaticraise (cont, [next_value_of_counter]),
                 L.lambda_unit))),
           L.lambda_unit)))
-  in lam
+  in env, lam
 
-let rec cps_non_tail env (lam : L.lambda) (k : Ident.t -> Ilambda.t)
+let let_cont_with_extra_params env ~continuation_on_stack ~is_exn_handler
```

Comment:
Rather than `continuation_on_stack`, I would use a name like `is_exn_handler`. It would make it easier to read the call sites.

Reply:
Sorry, what I meant is that it should be redundant with `is_exn_handler` (which is already there)

Reply:
And here I thought it was a sharp jab, ahah. You're right, it is redundant, I'll remove it.